src/test/java/com/artipie/rpm/meta/XmlOthersTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import com.jcabi.matchers.XhtmlMatchers;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Testcase for {@link XmlOthers} class.
+ * @since 0.1
+ * @todo #80:30min add another test for adding a changelog
+ */
+public class XmlOthersTest {
+
+    /**
+     * Test package added successfully.
+     * @param tmp Temp path
+     * @throws Exception
+     * @checkstyle
+     */
+    @Test
+    void canAddPackage(@TempDir final Path tmp) throws Exception {
+        final Path xml = tmp.resolve("others.xml");
+        try (XmlOthers others = new XmlOthers(xml)) {
+            others.startPackages();
+            others.addPackage(
+                "aom", "aarch64",
+                "7ae"
+            ).close();
+        }
+        MatcherAssert.assertThat(
+            new String(Files.readAllBytes(xml), StandardCharsets.UTF_8),
+            // @checkstyle LineLengthCheck (1 line)
+            XhtmlMatchers.hasXPath("/*[local-name()='otherdata']/*[local-name()='package' and @name='aom' and @pkgid='7ae' and @arch='aarch64']")
+        );
+    }
+
+    /**
+     * Test adding a version for the package.
```

Comment:
@ammaratef45 just to let you know: checkstyle doesn't force writing javadocs for test methods, so you can remove it if you want